weave-js/src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,246 @@
+import * as RadixDropdownMenu from '@radix-ui/react-dropdown-menu';
```

Comment:
this file is copied from [core](https://github.com/wandb/core/blob/6a43c44dbe43309021445aeb8bf60f2b0992ab0b/frontends/app/src/components/DropdownMenu/DropdownMenu.tsx)